source/draw/Color.ooc

```diff
@@ -137,6 +142,31 @@ ColorYuv: cover {
 	operator != (other: ColorBgra) -> Bool { !this equals(other) }
 }
 
+ColorYuva: cover {
+	yuv: ColorYuv
+	alpha: UInt8
+	init: func@ (=yuv, =alpha)
+	init: func@ ~uint8 (y, u, v, a: UInt8) {
+		this yuv = ColorYuv new(y, u, v)
+		this alpha = a
+	}
+	init: func@ ~default { this init(0, 128, 128, 255) }
+	init: func@ ~int (y, u, v, a: Int) { this init(y as UInt8, u as UInt8, v as UInt8, a as UInt8) }
```

Comment:
Do you actually need this constructor at all? I think rock might correctly match the `~uint8` constructor and implicitly cast if you used `Int` arguments.